lib/handler/RedirectHandler.js

```diff
@@ -91,6 +92,15 @@ class RedirectHandler {
       ? null
       : parseLocation(statusCode, headers)
 
+    if (this.opts.throwOnMaxRedirect && this.history.length >= this.maxRedirections) {
+      if (this.request) {
+        this.request.abort()
```

Comment:
Let's also abort it with the same error as we are aborting this `RedirectHandler`
```suggestion
        this.request.abort(new Error('max redirects')
```

**Optional**
- You can maybe create an error within [Errors](https://github.com/nodejs/undici/blob/main/lib/core/errors.js) to have a fully fledge undici error

Reply:
thank you very much I will try to create a full thank you error message today @metcoder95